backend/src/nodes/nodes/onnx/load_model.py

```diff
@@ -24,7 +24,7 @@ def __init__(self):
         )
         self.inputs = [OnnxFileInput(primary_input=True)]
         self.outputs = [
-            OnnxModelOutput(),
+            OnnxModelOutput(kind="onnx"),
```

Comment:
Why does this not broadcast?

Reply:
Because when I was trying to figure out how to do the generic output + tag, I thought should_broadcast was related to the model preview layout, so I'd tried removing it. Once I got it working, I didn't add it back in. Everything works without it, including the pre-connection validation, so I honestly don't really know what it's meant for? Could you explain that, because it doesn't seem like it's doing what I thought it did here.

Reply:
This shouldn't work. Without broadcasting, the frontend cannot receive the information necessary to narrow down types.

Reply:
¯\\\_(ツ)_/¯

Not sure what to tell you. I don't have that in, and you can see it's working.

Reply:
Now, I know why. You implemented  `get_broadcast_data` incorrectly. The output is supposed to decide itself whether it broadcasts by returning either `None` or a value. [Look at the NCNN implementation](https://github.com/chaiNNer-org/chaiNNer/blob/a635cf2a111ddd7382de8017f921ad5d71aa7807/backend/src/nodes/properties/outputs/ncnn_outputs.py#L18). So the ONNX output always broadcasts because it completely ignores `should_broadcast`, which is incorrect.